runtime/moonbase/src/lib.rs

```diff
@@ -1255,7 +1264,26 @@ impl Default for AssetType {
 
 impl From<MultiLocation> for AssetType {
 	fn from(location: MultiLocation) -> Self {
-		Self::Xcm(location)
+		match location {
+			// Change https://github.com/paritytech/cumulus/pull/831
+			// This avoids interrumption once they upgrade
+			// We map to the previous location so that the assetId is well calculated
+			// TODO: we might want to do a migration after statemine has upgraded
```

Comment:
We should remove this TODO and implement the mapping in reverse (support new location as default, and mapping of old location) and the migration pre-fix and not post-fix as written in the todo

Reply:
sounds good, I am on it right now